tests/unittest/benchmark_runner/common/template_operations/golden_files/func_ci_vdbench_kata_ODF_PVC_False/vdbench_pod.yaml

```diff
@@ -65,8 +67,15 @@ spec:
           value: "3"
         - name: SIZE_PER_FILE # size in MB
           value: "10"
+        #state-signals
+        - name: REDIS_HOST
+          value: ""
+        - name: WORKLOAD_METHOD
+          value: "/vdbench/vdbench_runner.sh"
+        - name: TIMEOUT
+          value: "3600"
       command: ["/bin/bash"]
-      args: ["-c", "/vdbench/vdbench_runner.sh"]
+      args: ["-c", "$WORKLOAD_METHOD"]
```

Comment:
Just use $WORKLOAD_METHOD as the command.

Reply:
What do you mean ?

Reply:
Never mind, I can see why you might need to do it this way here.